substra/cli/interface.py

```diff
@@ -193,7 +230,7 @@ def cli(ctx):
     For help using this tool, please open an issue on the Github repository:
     https://github.com/SubstraFoundation/substra-cli
     """
-    pass
+    ctx.obj = GlobalConf()
```

Comment:
I'm always wary of very generic names, I always prefer to have an explicit one. I understand that using `global_conf` may be a bit long, so what do you think of using `conf` instead?

Reply:
Unfortunately, we cannot use another attribute's name, it is specified in the documentation that we must use `obj` to store these data. I tried to, but it doesn't worked :c 
`obj – an arbitrary object of user data.`

Link: https://click.palletsprojects.com/en/7.x/api/#click.Context

Reply:
Oh thank you, I wasn't aware of this default `obj` option. Let's use this then, this is clearly the way things should be done.